### PR TITLE
Video support for Yival

### PR DESCRIPTION
**Pull request by zetianluo**
For the task 7, supporting video feature for Yival

### CODE REVIEW

File: src/yival/demo/headline_generation.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ def headline_generation(
     )
     res = MultimodalOutput(
         text_output=response['choices'][0]['message']['content'],
+        video_output=["https://www.w3schools.com/html/mov_bbb.mp4"]
```
Comment (crazycth) [inline src/yival/demo/headline_generation.py:53]: why hard code the video_output here

Comment (zetianluo) [reply src/yival/demo/headline_generation.py]: Removed

File: src/yival/cli/run.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ def add_arguments_to(subparser):
     parser.add_argument(
         "--interactive",
         action="store_true",
-        default=False,
+        default=True,
```
Comment (crazycth) [inline src/yival/cli/run.py:39]: Why set interactive=True here 🤔

**Comment by crazycth** [comment]
Could you provide an entire custom_func + yaml config to show video case with replicate?

Maybe we will have a colab and post later~

**Comment by zetianluo** [comment]
> Could you provide an entire custom_func + yaml config to show video case with replicate?
> 
> Maybe we will have a colab and post later~

I have a colab, I will post later for review!

File: src/yival/cli/run.py (hunk 2 above)
Comment (zetianluo) [reply src/yival/cli/run.py]: Good catch, I forgot to change back when I was testing. Reseted

**Review by crazycth — APPROVED** [review]

**Pull request merged**